spec/REPORT.md

```diff
@@ -0,0 +1,87 @@
+# Terms
+
+* Test is a single functionality check.
+    * Properties:
+        * name;
+        * package name;
+        * class name;
+        * duration;
+        * log;
+        * device name;
```

Comment:
It's more abstract thing than device and combines both emulators and real devices